src/MAUI/Maui.Samples/Samples/Data/EditFeatureAttachments/EditFeatureAttachments.xaml.cs

```diff
@@ -134,17 +134,19 @@ private async void AddAttachment_Click(object sender, EventArgs e)
                 string filename;
 
                 // Microsoft.Maui.Storage.FilePicker shows the iCloud picker (not photo picker) on iOS.
-                // This iOS code shows the photo picker.
+                
 #if IOS || MACCATALYST
-                Stream imageStream = await GetImageStreamAsync();
-                if (imageStream == null)
+                attachmentData = await GetImageBytesAsync();
+
+                // Return if no image was loaded.
+                if (attachmentData == null) return;
```

Comment:
I like this style of having one-line if statements without { } ... maybe at tomorrow's meeting we can update this coding convention on the wiki page because currently the convention is having { }.